_tools/release_docs/main.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
```

Comment:
I think it would be good to add a comment at the top saying what all it does right now. like change .md to .html, add "layout: docwithnav" ....